### PR TITLE
T/16960 Fix paste message in other locales

### PR DESCRIPTION
**Pull request by f1ames**
Renamed the lang entry from `pasteMsg` to `pasteDefaultMsg`.

### CODE REVIEW

File: dev/langtool/meta/ckeditor.plugin-clipboard/meta.txt [hunk 1]
```diff
@@ -6,4 +6,4 @@ copyError = A notification message shown when browser is blocking copying, and u
 cut = Toolbar button tooltip for the Cut feature.
 cutError = A notification message shown when browser is blocking cutting, and user should use the keyboard instead. %1 sequence is replaced with a keystroke that, when pressed, will perform the action.
 paste = Toolbar button tooltip for the Paste feature.
-pasteMsg = A notification message shown when browser is blocking pasting, and user should use the keyboard instead. %1 sequence is replaced with a keystroke that, when pressed, will perform the action.
+pasteDefaultMsg = A notification message shown when browser is blocking pasting, and user should use the keyboard instead. %1 sequence is replaced with a keystroke that, when pressed, will perform the action.
```
Comment (mlewand) [inline dev/langtool/meta/ckeditor.plugin-clipboard/meta.txt]: I'd rename it simply to pasteNotification, though longer by 2 chars, it has just 2 words and reflects better what it refers to.

Comment (mlewand) [inline dev/langtool/meta/ckeditor.plugin-clipboard/meta.txt]: Also please rename the same lang property in `pastetext` plugin, so they're consistent.

File: tests/plugins/clipboard/_helpers/pasting.js [hunk 2]
```diff
@@ -169,7 +169,7 @@ function testResetScenario( editor, queue ) {
 function getDefaultNotification( editor, command, keyInfo ) {
 	var keystroke = keyInfo || CKEDITOR.tools.keystrokeToString( editor.lang.common.keyboard,
 		editor.getCommandKeystroke( editor.commands[ command ] ) ),
-		msg = editor.lang[ command === 'paste' ? 'clipboard' : command ].pasteMsg
+		msg = ( command === 'paste' ? editor.lang.clipboard.pasteDefaultMsg : editor.lang[ command ].pasteMsg )
```
Comment (mlewand) [inline tests/plugins/clipboard/_helpers/pasting.js]: `editor.lang[ command ].pasteMsg` should be also reflected with property change.

**Comment by f1ames** [comment]
The `pasteNotification` lang entry added.

**Comment by mlewand** [comment]
Rebased the branch, and skipping changelong entry to avoid the noise.

**Pull request merged**